employee-rostering-backend/src/test/java/org/optaweb/employeerostering/generator/RosterGeneratorTest.java

```diff
@@ -87,69 +94,55 @@
         return restTemplate.exchange(tenantPathURI, HttpMethod.GET, null,
                                      new ParameterizedTypeReference<List<Tenant>>() {});
     }
+    
+    @Before
+    public void setup() {
+        tenantId = rosterGenerator.generateRoster(2, 7).getTenantId();
+    }
 
```

Comment:
We should have a `@After` method that deletes the generated rosters so that the rosters don't remain after the tests.